filecoin-hashers/src/poseidon.rs

```diff
@@ -584,18 +571,9 @@ mod tests {
 
         let circuit_hashed = PoseidonFunction::hash_md_circuit(&mut cs, circuit_data.as_slice())
             .expect("hash_md_circuit failure");
-        let hashed_alloc =
-            &num::AllocatedNum::alloc(cs.namespace(|| "calculated"), || Ok(hashed_fr))
-                .expect("alloc failure");
-        constraint::equal(
-            &mut cs.namespace(|| "enforce correct"),
-            || "correct result",
-            &hashed_alloc,
-            &circuit_hashed,
-        );
 
         assert!(cs.is_satisfied());
-        let expected_constraints = 2_771;
+        let expected_constraints = 2_770;
```

Comment:
What happened here?

Reply:
I removed one additional check that was adding a constraint